bin/custom/bookstore/bookstorecore/resources/bookstorecore-items.xml

```diff
@@ -131,6 +133,10 @@
 					<modifiers optional="true" />
 					<persistence type="property" />
 				</attribute>
+				<attribute qualifier="mostRented" type="java.lang.String">
+					<modifiers optional="true" />
+					<persistence type="property" />
+				</attribute>
 			</attributes>
```

Comment:
In genere ogni volta che aggiungiamo un nuovo attributo ne definiamo anche la localizzazione in modo da visualizzarlo correttamente nell'etichetta del backoffice. Non vedo questa localizzazione nella pull request